ynr/apps/elections/uk/tests/ee_import_results.py

```diff
@@ -4,7 +4,7 @@
     """
 {
   "count": 518,
-  "next": "https://elections.democracyclub.org.uk/api/elections/?current=True&limit=100&offset=100",
+  "next": "https://elections.democracyclub.org.uk/api/elections/?poll_open_date__gte=fakedate&limit=100&offset=100",
```

Comment:
Since doing this, I found I needed to use [freezegun](https://github.com/spulec/freezegun) in https://github.com/DemocracyClub/WhoCanIVoteFor/pull/462 which is actually a nicer way of doing this. I might actually go back and revisit this commit and use that for the date mocking before we merge it but I was keen to try and get PRs open for all this by the end of the day so I haven't done it yet.